pkgs/development/compilers/gradient-halide/default.nix

```diff
@@ -0,0 +1,62 @@
+{ llvmPackages, lib, fetchFromGitHub, cmake
+, libpng, libjpeg, mesa_noglu, eigen3_3, openblas
+}:
+
+llvmPackages.stdenv.mkDerivation rec {
+
+  pversion = "2018-08-31";
+  name = "gradient-halide-${builtins.replaceStrings ["_"] ["."] pversion}";
+
+  src = fetchFromGitHub {
+    owner = "jrk";
+    repo = "gradient-halide";
+    rev = "f630dd26eb141349d147f20ace39793d5bc717d2";
+    sha256 = "0qhk1plc4697p1livf9xz2c5aqaq6v9sx1pr0ia6x7sjkmggc9rc";
+  };
+
+  patches = [ ./nix.patch ];
```

Comment:
Do you think it would be possible to upstream some of these patches?

Reply:
Perhaps, but I didn't make these patches myself so I don't know exactly what would we involved.  However, it looks like I might soon create an issue/PR against the stock `halide` NIx package so perhaps I'll suggest it there.